alembic/versions/27622169eb7d_change_user_role_id_foreign_key_to_a_.py

```diff
@@ -0,0 +1,34 @@
+"""Change User role_id foreign key to a role_name foriegn key
+
+Revision ID: 27622169eb7d
+Revises: 6d647b5d5baa
+Create Date: 2023-08-24 15:54:08.386098+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '27622169eb7d'
+down_revision = '6d647b5d5baa'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('user_account', sa.Column('role_name', sa.Text(), nullable=False))
```

Comment:
I don't quite understand what this is doing. Can you help me understand?

Reply:
This migration is just dropping the `role_id` column and its foreign key constraint and adding in a `role_name` column with a foreign key constraint. Both are unique, but using the name is more convenient

Reply:
Oh gotcha. I was reading it wrong, thinking that it was adding a `user_account` column and then dropping it 3 lines later, but it's not. Thanks